roles/lib/files/FWO.Api.Client/JsonStringConverter.cs

```diff
@@ -0,0 +1,79 @@
+﻿using FWO.Api.Data;
```

Comment:
can you please explain this magic/generic method - could this be used in other context as well?